velox/common/base/VeloxException.h

```diff
@@ -36,6 +36,9 @@ DECLARE_int32(velox_exception_system_stacktrace_rate_limit_ms);
 
 namespace facebook {
 namespace velox {
+static const char* kNullKeyErrorMessage = "map key cannot be null";
```

Comment:
These are Presto-specific.... probably do not belong here.

Reply:
I can move them